src/bp/ui-admin/src/Pages/Server/Alerting.tsx

```diff
@@ -89,17 +105,22 @@ class Alerts extends Component {
 
   renderNoData() {
     return (
-      <Jumbotron>
-        <Row>
-          <Col style={{ textAlign: 'center' }} sm="12" md={{ size: 8, offset: 2 }}>
-            <h1>
-              <IoIosArchive />
-              &nbsp; Alerting is not enabled or there is no statistics.
-            </h1>
-            <p>Make sure that alerting is enabled in your Botpress Config.</p>
-          </Col>
-        </Row>
-      </Jumbotron>
+      <PageContainer title="Alerting & Incidents">
+        <Jumbotron>
+          <Row>
+            <Col style={{ textAlign: 'center' }} sm="12" md={{ size: 8, offset: 2 }}>
```

Comment:
Pretty sure there's a bootstrap & blueprint helper css class to center content

Reply:
Only left and right actually, but that page will need to be refactored when changes are applied to it